Open Judge System/Services/OJS.Services/Business/Contests/ContestsBusinessService.cs

```diff
@@ -172,10 +172,10 @@ public JudgeLoadResults CalculateLoadForContest(BaseContestBusinessModel model)
         {
             var responseModel = new JudgeLoadResults();
 
-            responseModel.PreviousAverageProblemRunTimeInSeconds = model.PreviousAverageProblemRunTimeInSeconds;
-            responseModel.PreviousContestExpectedProblems = model.PreviousContestExpectedProblems;
-            responseModel.PreviousContestSubmissions = model.PreviousContestSubmissions;
-            responseModel.PreviousContestParticipants = model.PreviousContestParticipants;
+            responseModel.PreviousAverageProblemRunTimeInSeconds = model.PreviousAverageProblemRunTimeInSeconds.Value;
```

Comment:
Are we sure it will have Value always?

Reply:
Yes, in case the contest is selected from the ComboxBox, the value will be calculated , in the other will, it will be added manually by the admin on the FE . In both ways we will have this value